spec/features/users/user_forgets_password.spec.rb

```diff
@@ -0,0 +1,8 @@
+scenario 'user forgets password, make new one' do
+  visit root
+  click_button 'Sign in'
+  click_link 'Forgot your password?'
+
+  expect(page).to have_content('Email')
+  expect(page).to have_content('Send me reset password instructions')
+end
```

Comment:
This will be a longer test later on that will include sending an email and whatnot. I'd remove this for now.